### PR TITLE
[REVIEW] Rename `skip_rows` parameter to `skiprows` in `read_parquet`, `read_avro` and `read_orc`

### PR DESCRIPTION
**Pull request by skirui-source**

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Review by galipremsagar — CHANGES_REQUESTED** [review]
We would have to make a docstring(present in `ioutils`) changes too now that this param has been renamed.

https://docs.rapids.ai/api/cudf/nightly/api.html?highlight=read_avro#cudf.io.avro.read_avro
https://docs.rapids.ai/api/cudf/nightly/api.html?highlight=read_avro#cudf.io.orc.read_orc
https://docs.rapids.ai/api/cudf/nightly/api.html?highlight=read_avro#cudf.io.parquet.read_parquet

**Review by isVoid — APPROVED** [review]
Make sure all related docstrings are updated. Can utilize the `find_in_files` function in VSCode to identify if there are any lines of `skip_rows` got left out. Otherwise lgtm. Great work digging into all the details!

**Comment by skirui-source** [comment]
> Make sure all related docstrings are updated. Can utilize the `find_in_files` function in VSCode to identify if there are any lines of `skip_rows` got left out. Otherwise lgtm. Great work digging into all the details!

So i just fidn the find-in-files and there's no search results for "skip_rows"... Does that mean that all docstrings have been updated?

**Comment by isVoid** [comment]
That would at least make sure the scope you searched has no strings that writes `skip_rows`. But yeah, would be a good indicator the strings are replaced.

**Comment by galipremsagar** [comment]
The current changes in the PR don't indicate doc changes. It could be possible that you did not commit and push the changes to your branch.

The doc changes I'm referring to are there following lines for the respective public API changes you made:

https://github.com/rapidsai/cudf/blob/branch-0.16/python/cudf/cudf/utils/ioutils.py#L33
https://github.com/rapidsai/cudf/blob/branch-0.16/python/cudf/cudf/utils/ioutils.py#L272
https://github.com/rapidsai/cudf/blob/branch-0.16/python/cudf/cudf/utils/ioutils.py#L133

**Comment by skirui-source** [comment]
> The current changes in the PR don't indicate doc changes. It could be possible that you did not commit and push the changes to your branch.
> 
> The doc changes I'm referring to are there following lines for the respective public API changes you made:
> 
> https://github.com/rapidsai/cudf/blob/branch-0.16/python/cudf/cudf/utils/ioutils.py#L33
> https://github.com/rapidsai/cudf/blob/branch-0.16/python/cudf/cudf/utils/ioutils.py#L272
> https://github.com/rapidsai/cudf/blob/branch-0.16/python/cudf/cudf/utils/ioutils.py#L133

I am just stuck on the builin

> The current changes in the PR don't indicate doc changes. It could be possible that you did not commit and push the changes to your branch.
> 
> The doc changes I'm referring to are there following lines for the respective public API changes you made:
> 
> https://github.com/rapidsai/cudf/blob/branch-0.16/python/cudf/cudf/utils/ioutils.py#L33
> https://github.com/rapidsai/cudf/blob/branch-0.16/python/cudf/cudf/utils/ioutils.py#L272
> https://github.com/rapidsai/cudf/blob/branch-0.16/python/cudf/cudf/utils/ioutils.py#L133

My issue is that I don't understand the "set up building environment" instructions to actually build the docs ...

File: CHANGELOG.md [hunk 1]
```diff
@@ -158,6 +158,7 @@
 - PR #6304 Fix span_tests.cu includes
 - PR #6331 Avoids materializing `RangeIndex` during frame concatnation (when not needed)
 - PR #6278 Add filter tests for struct columns
+- PR #6353 Renamed skip_rows parameter to skiprows
```
Comment (galipremsagar) [inline CHANGELOG.md:161]:
```suggestion
- PR #6353 Rename `skip_rows` parameter to `skiprows` in `read_parquet`, `read_avro` and `read_orc`
```

**Review by galipremsagar — APPROVED** [review]
This is now good to go!

**Pull request merged**